src/worker/exporters/nvidia_exporter.py

```diff
@@ -210,7 +211,7 @@ def jobID_update(self):
             gpuUuid = self.m_gpuIdToUUId[gpuId]
             gpuBusId = self.m_gpuIdToBusId[gpuId]
             gpuUniqueId = gpuUuid if dcgm_config['sendUuid'] else gpuBusId        
-            for fieldId in self.m_publishFieldIds:
+            for fieldId in self.m_publishFields[self.m_updateFreq]:
```

Comment:
Why is m_updateFreq used now?

Reply:
Python3 uses a dictionary vs the list python2 uses.

Reply:
python2 uses a list vs python3 using a dict